lib/matplotlib/stackplot.py

```diff
@@ -55,10 +55,7 @@ def stackplot(axes, x, *args, **kwargs):
     element in the stacked area plot.
     """
 
-    if len(args) == 1:
-        y = np.atleast_2d(*args)
-    elif len(args) > 1:
-        y = np.row_stack(args)
+    y = np.row_stack(args)
```

Comment:
No, it's `np.atleast_2d(*args)` (note the unpack) and *only* in the case where `len(args) == 1`.

Say `args` has shape (after casting to an array) `(1, x, y, ...)`.
- `np.atleast_2d(*args) == np.atleast_2d(<shape (x, y, ...)>) == <shape (x, y, ...)>`
- `np.row_stack(args)` also has shape (x, y, ...).

In the case `args` is 2D (shape (1, x)), both expressions result in a shape (1, x) as well.